app/views/layouts/application.html.erb

```diff
@@ -7,12 +7,13 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
```

Comment:
Consider moving the Google Fonts link to the head section of the HTML document for better performance.